ROADMAP.md

```diff
@@ -0,0 +1,39 @@
+# Jenkins Operator Roadmap
+
+This document outlines the vision and technical roadmap for [jenkinsci/kubernetes-operator](https://github.com/jenkinsci/kubernetes-operator) project.
+
+## Project Vision
+
+With Jenkins Operator project we want to enable our community to run Jenkins in cloud-native environments like Kubernetes, OpenShift and others. Also, support most of the public cloud providers (AWS, Azure, GCP) in terms of additional capabilities like backups, observability and cloud security.  
+
+
+With declarative configuration and full lifecycle management based on Operator Framework this can become the de facto standard for running Jenkins on top of Kubernetes.
+
+
+Now, we have a dedicated team which can bring more engineering effort to the project. We are now justified long term commitment for the OSS project (instead of working on best-effort basis as it was in the past).
```

Comment:
"We are now justified long term commitment for the OSS project". I'm not sure I understand. Can you elaborate what message you want to convey?